config/_default/config.toml

```diff
@@ -25,7 +25,7 @@ crowdfunding_url = "https://www.generosity.com/community-fundraising/make-a-more
 # Date and time formatting, see https://gohugo.io/functions/format/
 time_format_RFC3339 = "2006-01-02T15:04:05Z07:00"
 time_format_RFC822 = "Mon, 02 Jan 2006 15:04:05 -0700"
-time_format_default = "Jan 2, 2006"
+time_format_default = ":date_medium"
```

Comment:
I don't think this has worked as expected: https://deploy-preview-1340--letsencrypt.netlify.app/2019/10/09/onboarding-your-customers-with-lets-encrypt-and-acme.html

![image](https://user-images.githubusercontent.com/1465785/139712303-3fc3111e-3452-421b-8eb7-6e9007aaf430.png)

Reply:
Thanks for the catch, it should work as expected now!